ide/javascript2.debug/nbproject/project.xml

```diff
@@ -114,6 +114,7 @@
             <friend-packages>
                 <friend>org.netbeans.modules.debugger.jpda.js</friend>
                 <friend>org.netbeans.modules.debugger.jpda.truffle</friend>
+                <friend>org.netbeans.modules.java.lsp.server</friend>
```

Comment:
I don't see the reason for this change - am I missing the dependency relationship?

Reply:
Good catch. This is a change needed for the [java.lsp.server debugging](https://github.com/JaroslavTulach/netbeans/tree/java-lsp-server-debugging-attempt2), but not really a cleanup. Moreover one should bump module version when changing list of friends. 

Let's remove the change from this PR: 2847f6e